kinto_redis/permission.py

```diff
@@ -65,7 +65,9 @@ def remove_principal(self, principal):
     @wrap_redis_error
     def get_user_principals(self, user_id):
         user_key = 'user:%s' % user_id
-        return self._decode_set(self._client.smembers(user_key))
+        members = self._decode_set(self._client.smembers(user_key))
+        group_authenticated = self._decode_set(self._client.smembers('user:system.Authenticated'))
```

Comment:
nit: here we could add a comment that says *Fetch groups with authenticated among members*